docs/migration_0.29.md

```diff
@@ -0,0 +1,92 @@
+# Migration Guide from 0.28 to 0.29
+
+OpenTelemetry Rust 0.29 introduces few breaking changes. This guide aims to
```

Comment:
We have made a considerable number of breaking changes, right? We should use "a few" not "few".

```suggestion
OpenTelemetry Rust 0.29 introduces a few breaking changes. This guide aims to
```

Reply:
a few is correct yes! If there was no breaking change at all, then we didn't need migration guide too 🤣